appium/tests/helpers/AppiumHelper.ts

```diff
@@ -0,0 +1,19 @@
+import { CommonData } from "../data";
+
+class AppiumHelper {
+    /**
+     * Restart app with processArguments
+     */
+    static async restartApp() {
+        const bundleId = CommonData.bundleId.id;
+        await driver.terminateApp(bundleId);
+        const processArgs = ['--mock-fes-api', '--mock-attester-api'];
```

Comment:
This restarts app as a mock which may be surprising to someone who may wish to restart the normal app.

Let's pass processArgs as an argument of `restartApp` and then also let's save somewhere `const mockAppProcessArgs = ['--mock-fes-api', '--mock-attester-api']` so that we don't have to track two places where these are defined.

So you would call `AppiumHelper.restartApp(mockAppProcessArgs)` or something like that.

Reply:
ok, I agree